edalize/verilator.py

```diff
@@ -150,12 +122,41 @@ def _write_config_files(self):
         else:
             make_options = ''
 
-        with open(os.path.join(self.work_root, 'config.mk'), 'w') as config_mk:
-            config_mk.write(CONFIG_MK_TEMPLATE.format(
-                top_module        = self.toplevel,
-                vc_file           = self.verilator_file,
-                verilator_options = verilator_options,
-                make_options      = make_options))
+        self.render_template('verilator-config.j2',
+                             'config.mk',
+                             {
+                                'top_module'        : self.toplevel,
+                                'vc_file'           : self.verilator_file,
+                                'verilator_options' : verilator_options,
+                                'make_options'      : make_options
+                             })
+        commands = self.EdaCommands()
+        commands.add(['include', 'config.mk'], [], [])
+        # Not sure about this one,
+        # can verilator_root be set after running this script?
+        veril_root = os.getenv("VERILATOR_ROOT")
+        verilator_bin = ''
+        if veril_root:
+            verilator_bin = veril_root + '/bin/verilator'
+        else:
+            verilator_bin = 'verilator'
```

Comment:
Same here: Edalize have ability to execute script before running commands, that can e.g. setup environment, change conda env etc. Does this call is executed after this script?